project_categ_issue/__manifest__.py

```diff
@@ -1,40 +1,27 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (C) 2013 Daniel Reis
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# (c) 2013 Daniel Reis
+# (c) 2017 Rigoberto Martínez <rigo1985@gmail.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
 {
     'name': 'Per Project Configurable Categorie on Issues',
     'summary': 'Projects Issues can have an allowed category list',
-    'version': '8.0.0.1.0',
+    'version': '0.1.0',
     "category": "Project Management",
     'description': """\
-Adds to Issues the ability to limit selectable Categories to a Proeject's
+Adds to Issues the ability to limit selectable Categories to a Project's
 specific list.
 """,
-    'author': "Daniel Reis,Odoo Community Association (OCA)",
+    'author': 'Tecnativa, Odoo Community Association (OCA)',
```

Comment:
Thank you Rigoberto, IMO porting a module does not add enough originality in order to grant authorship and teh corresponding copyright.
Please add yourself to the contributors list in the README, but keep the original author in the manifest and code files.